packages/liveblocks-node/src/index.ts

```diff
@@ -90,8 +95,20 @@ export async function authorize(
   } catch (er) {
     return {
       status: 403,
-      body: 'Call to "https://liveblocks.io/api/authorize" failed. See "error" for more information.',
+      body: 'Call to "https://api.liveblocks.io/v2/rooms/{roomId}/authorize" failed. See "error" for more information.',
       error: er,
     };
   }
 }
+
+function buildLiveblocksAuthorizeEndpoint(
+  options: AllAuthorizeOptions,
+  room: string
```

Comment:
Can we rename this arg?

```suggestion
  roomId: string
```